compiler/x/codegen/OutlinedInstructions.cpp

```diff
@@ -175,18 +120,34 @@ void TR_OutlinedInstructions::generateOutlinedInstructionsDispatch()
    if (_targetReg)
       {
       TR_ASSERT(resultReg, "assertion failure");
+      TR_ASSERT(resultReg->getKind() == _targetReg->getKind(), "OutlinedInstructions: targetReg must have same time as resultReg.");
```

Comment:
Do you mean "same kind" rather than "same time" ?

Reply:
Good catch, it has been updated.